index.html

```diff
@@ -230,6 +230,25 @@ <h3 class="intro2">LET’S CONNECT</h3>
             <li class="line"></li>
         </ul>
     </section>
+    <section class="contact-me">
+        <div class="bg-image">
+            <h1 class="contact-title">Contact me</h1>
+            <p class="contact-text">
+                If you have an application you are interested in developing, a feature
+                that you need built or a project that needs coding. I’d love to help
+                with it
+            </p>
+            <form action="https://formspree.io/f/xqkgeweb" method="POST" class="contact-form">
+                    <input type="text" name="name" id="username" placeholder="Your Name" maxlength="30" required />
+                <input type="email" name="email" id="useremail" placeholder="Your Email" required />
+                <textarea name="message" id="usermessage" cols="30" rows="10" maxlength="500"
+                    placeholder="Write your message here"></textarea>
+                <div class="button-resume btn-form">
```

Comment:
- [ ] Great job making the email and name input required, it's one of the requirements to make all inputs required, Kindly add that.